simple/stats/nodes.py

```diff
@@ -262,6 +262,10 @@ def entities_with_type(self, entity_dcids: list[str], entity_type: str):
     for entity_dcid in entity_dcids:
       self.entity_with_type(entity_dcid, entity_type)
 
+  def entities_with_types(self, dcid2type: dict[str, str]):
```

Comment:
Nit: add a docstring (at first glance it wasn't clear to me this function mutated the object state)

Reply:
Done.